bin/trendAnalyse.sh

```diff
@@ -244,6 +244,84 @@ function processProjectToDB() {
 
 }
 
+function processRNAProjectToDB {
+	local _rnaproject="${1}"
+	local _processrnaprojecttodb_controle_line_base="${2}"
+	
+	CHRONQC_TMP="${TMP_TRENDANALYSE_DIR}/tmp/"
+	CHRONQC_RNAPROJECTS_DIR="${TMP_TRENDANALYSE_DIR}/RNAprojects/${_rnaproject}/"
+	CHRONQC_DATABASE_NAME="${TMP_TRENDANALYSE_DIR}/database/"
+
+	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Removing files from ${CHRONQC_TMP} ..."
+	#rm -rf "${CHRONQC_TMP:-missing}"/*
```

Comment:
deze regel staat uitgecommentarieerd, er wordt nu dus niks daadwerkelijk verwijderd, is dat de bedoeling?